aeron-client/src/main/java/io/aeron/ChannelUriStringBuilder.java

```diff
@@ -2171,6 +2172,15 @@ public ChannelUriStringBuilder maxResend(final Integer maxResend)
         return this;
     }
 
+    public int streamId(){
+        return streamId;
+    }
+
+    public void streamId(int streamId)
+    {
+        this.streamId = streamId;
+    }
+
```

Comment:
These should return and accept and `Integer` object. Also you need a version that reads `stream-id` from a `ChannelUri` (e.g. see `ChannelUriStringBuilder#termOffset(io.aeron.ChannelUri)`). And it needs to filled in the constructor of the class.

Reply:
I have fixed the Integer part. I forget to change the signatures of the methods after converting streamId from int to Integer.

But the streamId isn't part of the channelUri; you can't create a channel like this:

```
Publication publication = aeron.addPublication( "aeron:udp?endpoint=localhost:40123|streamId=10");
```

Exposing the property as PARAM_NAME and adding it to the ChannelURI, could lead to end users getting confused. WDYT.

Reply:
We want to make this parameter official. It should be possible to set `stream-id` in the channel uri and will be adding validation that the `stream-id` parameter is the same as `streamId` argument to the method.

Reply:
ok.